.eslintrc

```diff
@@ -49,32 +58,60 @@
         "no-unused-expressions": 2,
         "no-with": 2,
         "radix": 2,
+        "vars-on-top": 1,
```

Comment:
I used this revently in several projects and at the beginning it was quite annoying, if you are not used to it :wink: but I agree that this can reduce confusing code.